lookout/style/typos/preparation.py

```diff
@@ -109,37 +113,51 @@ def prepare_data(config: Optional[Mapping[str, Any]] = None) -> pandas.DataFrame
     if raw_data_path is None or not os.path.exists(raw_data_path):
         raw_data_path = os.path.join(config["data_dir"],
                                      config["raw_data_filename"])
+        log.info("Raw dataset is not found. Downloading to %s from %s",
+                 raw_data_path, config["dataset_url"])
         _download_url(config["dataset_url"], raw_data_path)
 
     data = pandas.read_csv(raw_data_path, index_col=0, keep_default_na=False)
+    log.debug("Raw dataset shape: %s", data.shape)
     if config["frequency_column"] not in data.columns:
+        log.info("Frequency column is not found. Set all frequencies to 1.")
         data[Columns.Frequency] = 1
     else:
+        log.info("Frequency column `%s` is found.", config["frequency_column"])
         data = data.rename(columns={config["frequency_column"]: Columns.Frequency})
 
     # Expand dataframe by splits (repeat rows for every token in splits)
     data[Columns.Split] = data[Columns.Split].astype(str)
+    log.debug("Expand data by splits")
     flat_data = flatten_df_by_column(data, Columns.Split, Columns.Token,
                                      apply_function=lambda x: x.split())
+    log.debug("Expanded data shape %s", flat_data.shape)
 
-    # Collect statistics for tokens
+    log.info("Collect statistics for tokens")
     stats = flat_data[[Columns.Frequency, Columns.Token]].groupby([Columns.Token]).sum()
     stats = stats.sort_values(by=Columns.Frequency, ascending=False)
 
-    # Derive new vocabulary for future use
+    log.info("Derive new vocabulary")
     frequencies_tokens = set(stats.index[:(config["frequencies_size"] or len(stats))])
+    log.info("Tokens with frequencies data size: %d", len(frequencies_tokens))
     vocabulary_tokens = set(stats.index[:config["vocabulary_size"]])
-    print_frequencies(vocabulary_tokens, stats, os.path.join(
-        config["data_dir"], config["vocabulary_filename"]))
-    print_frequencies(frequencies_tokens, stats, os.path.join(
-        config["data_dir"], config["frequencies_filename"]))
+    log.info("vocabulary size: %d", len(vocabulary_tokens))
```

Comment:
The logs should all start with a capital letter or with a small letter

Reply:
@zurk Missed this ^

Reply:
All logs start with small letter :)

Reply:
Oh 😄